packages/x-components/static-docs/build-search-ui/web-archetype-integration-guide.md

```diff
@@ -224,7 +225,7 @@ following configuration options:
 | Name                                                   | Type                                                                                 | Description                                                                                                                                                                                                                                                                                      |
```

Comment:
Name of what? What do you really mean: Name or Parameter?

Reply:
Name of the parameter...haha, we may change it to `Paramater`

Reply:
Changed to parameter, as we has in old docs, makes sense